config/spryk/spryks/Spryker/Glue/BackendApi/Plugin/AddGlueBackendApiResourcePlugin.yml

```diff
@@ -0,0 +1,90 @@
+spryk: template
+description: "Adds a new GlueBackendApi Resource Plugin."
+mode: both
+
+arguments:
+    module:
+        inherit: true
+
+    organization:
+        inherit: true
+        default: Spryker
+
```

Comment:
We added everywhere `application` please also add here.

Reply:
Please also add:

```
apiType:
        default: JSON
```

Reply:
Application is already on line 34.